api/v1alpha1/nodepool_types.go

```diff
@@ -18,6 +18,7 @@ const (
 	NodePoolUpdateManagementEnabledConditionType = "UpdateManagementEnabled"
 	NodePoolAutoscalingEnabledConditionType      = "AutoscalingEnabled"
 	NodePoolReadyConditionType                   = "Ready"
+	NodePoolReconciliationActive                 = "ReconciliationActive"
```

Comment:
can we NodePoolReconciliationActive -> NodePoolReconciliationActiveConditionType?
just to be consistent with all the others here.